### PR TITLE
Merge pull request #4 from RickWieman/patch-1

### PR DESCRIPTION
**Pull request by crrood**
Add Payout to service_url_base

### CODE REVIEW

**Comment by crrood** [comment]
Hi @rikterbeek,

Updated to version 1.0.2.

**Comment by crrood** [comment]
Also renamed the gem to 'adyen-official' to avoid a conflict with the existing 'adyen' library on rubygems.org

**Comment by crrood** [comment]
Fixed the issue with live checkout URLs, changed name to adyen-ruby-api-client to be consistent with the other libraries

File: lib/adyen/client.rb [hunk 1]
```diff
@@ -33,10 +34,15 @@ def service_url_base(service)
       else
         case service
         when "Checkout"
-          "https://checkout-#{@env}.adyen.com"
+          if @env == :test
+            "https://checkout-#{@env}.adyen.com"
```
[the remark this answers is not in the document]
Comment (crrood) [reply lib/adyen/client.rb]: Sure, makes sense.  I've committed the change.

**Comment by rikterbeek** [comment]
Hi @crrood,

This is good for the quickfix let's release it.
I will be in SF from next week so let's sit down to have the same approached realised as the other libs.

Regards,
Rik
Adyen

**Pull request merged**